src/apyfixedarray.cc

```diff
@@ -871,6 +871,58 @@ std::string APyFixedArray::repr() const
     return ss.str();
 }
 
+APyFixedArray APyFixedArray::reshape(nb::tuple new_shape) const
+{
+    std::vector<std::size_t> new_shape_vec;
```

Comment:
One may consider moving the argument checking to a separate function that can be used by both float and fixed, but that may be a later change (not sure where to put it though, create `array_utils.h`?).